src/eureka/S3_data_reduction/optspex.py

```diff
@@ -784,7 +784,9 @@ def optimize(meta, subdata, mask, bg, spectrum, Q, v0, p5thresh=10,
                             plt.figure(3803)
                             plt.clf()
                             plt.suptitle(str(i) + "/" + str(nx))
-                            plt.plot(subdata[:, i], 'bo')
+                            plt.errorbar(np.arange(ny), subdata[:, i],
+                                         np.sqrt(variance[:, i]),
```

Comment:
Does this end up being xerr or yerr? I believe the ordering of the keywords is xerr first, but I might be wrong. I always specify `yerr=` since I can't remember which error it assumes you've input if you only input one error